source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1214,6 +1216,38 @@ def _get_rowHeaderText(self):
 	def _get_columnHeaderText(self):
 		return self._tableHeaderTextHelper("column")
 
+	def _get_selectionContainer(self):
+		if self.table:
+			return self.table

```

Comment:
This implicitly returns None when not self.table. I would prefer an explicit return of None or even better, super below this line, though not strictly necessary. I leave it up to you.